### DISCUSSION TITLE
Require information on web driver IO

### DISCUSSION THREAD

**Discussion by nikhithavangaru** [webdriverio/webdriverio#8463] [General]
Hi

I am Lakshmi Nikhitha working at one of the big reputed companies, I would like to know few things that,

1. Can we expect migration support for web driver IO? 
2. Is it an open source or licensed tool?
3. If it is a licensed tool, how much does it cost for 10-15 people?

Please let me know for any further information.

Thanks & Regards
Lakshmi Nikhitha Vangaru

**Comment by christian-bromann** [comment]
> Can we expect migration support for web driver IO?

There are some folks or work as WebdriverIO freelancer and might be able to help out.

> Is it an open source or licensed tool?

Yes, WebdriverIO is MIT licensed.

> If it is a licensed tool, how much does it cost for 10-15 people?

No cost!
